cmd/manager/main.go

```diff
@@ -47,6 +48,8 @@ func main() {
 	// controller-runtime)
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 
+	pflag.StringSliceVarP(&products, "products", "p", []string{"all"}, "--products=rhsso,fuse")
```

Comment:
Can we store `"all"` as a var somewhere, as we use it twice here and it could be used more in the future?

Reply:
yes will do it in my next pr